include/z64scene.h

```diff
@@ -152,14 +152,14 @@ typedef union RoomShape {
     RoomShapeCullable cullable;
 } RoomShape; // "Ground Shape"
 
-typedef enum RoomBehaviorType1 {
-    /* 0 */ ROOM_BEHAVIOR_TYPE1_0,
-    /* 1 */ ROOM_BEHAVIOR_TYPE1_1,
-    /* 2 */ ROOM_BEHAVIOR_TYPE1_2,
-    /* 3 */ ROOM_BEHAVIOR_TYPE1_3, // unused
-    /* 4 */ ROOM_BEHAVIOR_TYPE1_4, // unused
-    /* 5 */ ROOM_BEHAVIOR_TYPE1_5
-} RoomBehaviorType1;
+typedef enum RoomType {
+    /* 0 */ ROOM_TYPE_FIELD,
+    /* 1 */ ROOM_TYPE_DUNGEON, // Blocks Sun's Song's time advance effect.
+    /* 2 */ ROOM_TYPE_INDOORS, // Reduces player run speed and blocks player from attacking or jumping.
+    /* 3 */ ROOM_TYPE_TEST_NO_COLOR_DITHERING, // Unused. Color dithering is turned off when drawing the room and other things
+    /* 4 */ ROOM_TYPE_4, // unused. Prevents switching to CAM_SET_HORSE when mounting a horse.
```

Comment:
Why `TEST` for `ROOM_TYPE_TEST_NO_COLOR_DITHERING`? I wonder if we should give up and call it `ROOM_TYPE_3` like the one below, since it doesn't fit with the others, there could have been some environment (like desert, idk) where it makes sense to turn off color dithering. Similar to how `ROOM_TYPE_BOSS` only adjust the lights but we don't call it `ROOM_TYPE_TEST_NO_ADJUST_LIGHTS`.

Reply:
think i agree on just unnaming it. am curious to see a image comparison of what it looks like though

Reply:
I don't know how accurate Simple64 is, but you can see a significant difference on highly stretched textures

![RoomType3_A](https://github.com/user-attachments/assets/bf0d2398-6683-4a57-89f8-bd62c3707281)
![RoomType3_B](https://github.com/user-attachments/assets/486eac01-f256-4e6f-b4ea-5ebe13a80fa4)

![RoomType3_C](https://github.com/user-attachments/assets/864cfa31-2988-41da-a0df-74a951f9ff7e)
![RoomType3_D](https://github.com/user-attachments/assets/bd05fe1e-c9b9-4516-a891-519c135e727a)

Reply:
Thanks, thats interesting.
Anyway yeah I think we can unname it at least for the time being. Feel free to comment on the dither thing though